### PR TITLE
Displays scoring from the API for each google search result.

### PR DESCRIPTION
**Pull request by Bracciata**

### CODE REVIEW

File: extension/background.js [hunk 1]
```diff
@@ -1,31 +1,56 @@
...
-// Node.js APIs. The below event listener registration can be broken in to 4 distinct parts:
-//
-// * chrome      - the global namespace for Chrome's extension APIs
-// * runtime     – the namespace of the specific API we want to use
-// * onInstalled - the event we want to subscribe to
-// * addListener - what we want to do with this event
-//
-// See https://developer.chrome.com/docs/extensions/reference/events/ for additional details.
+
+async function getA11yScores() {
```
Comment (emma-hubka) [inline extension/background.js:2]: getHa11yScores

File: extension/background.js [hunk 2]
```diff
@@ -1,31 +1,56 @@
...
+          console.log(err);
+        } 
+      }
+      }
+    }
+  }
+
+
+chrome.webRequest.onCompleted.addListener(function (details) {
+  // Run the get A11y Scores function after google searching.
```
Comment (emma-hubka) [inline extension/background.js:42]: Ha11y

File: extension/background.js [hunk 3]
```diff
@@ -1,31 +1,56 @@
...
+  chrome.scripting.executeScript({
+    target: { tabId: details.tabId },
+    function: getA11yScores
```
Comment (emma-hubka) [inline extension/background.js:45]: Ha11y

File: extension/manifest.json [hunk 4]
```diff
@@ -1,9 +1,18 @@
 {
-  "name": "Hello, World!",
-  "version": "1.0",
+  "name": "A11ySearchScores",
```
Comment (emma-hubka) [inline extension/manifest.json]: Ha11y

File: extension/manifest.json [hunk 5]
```diff
@@ -1,9 +1,18 @@
...
+  "action": {},
   "manifest_version": 3,
+  "version": "0.1",
+  "description": "Displays the A11y score of sites when searching in google.",
```
Comment (emma-hubka) [inline extension/manifest.json:6]: Ha11y

File: extension/background.js [hunk 6]
```diff
@@ -1,31 +1,56 @@
...
+          try{
+          fetch(`http://localhost:8080?url=${encodeURIComponent(linkElement.href)}`).then((results) => {
+            return results.json();
+          }).then(json => {
+            if (json.issues) {
+              // Temporary hack for displaying the results from the API
+              score.innerHTML = `Score: ${json.issues} errors`;
+            } else {
+              // This should never happen but we should be prepared to handle this.
+              score.innerHTML = `Pa11y could not process this link - Error was ${json.error}`;
```
Comment (emma-hubka) [inline extension/background.js]: Ha11y

**Review by emma-hubka — CHANGES_REQUESTED** [review]

File: extension/manifest.json (hunk 5 above)
Comment (Bracciata) [reply extension/manifest.json:6]: No fix needed

**Pull request merged**